src/test/regress/isolation_schedule

```diff
@@ -108,3 +108,7 @@ test: isolation_acquire_distributed_locks
 test: isolation_concurrent_move_create_table
 
 test: isolation_check_mx
+
+# MERGE
```

Comment:
I think `test: isolation_check_mx` is intended to be the last test in the file. Can you maybe move merge tests above that?